test/fakeintake/server/server.go

```diff
@@ -68,7 +68,8 @@ type Server struct {
 	urlMutex sync.RWMutex
 	url      string
 
-	store serverstore.Store
+	storeDriver string
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would use a typed string, something like
```golang
type storeDriver string

const (
  inMemory storeDriver = "inmemory",
  sql storeDriver = "sql"
)
```

Reply:
Can be addressed in a follow up